NAMESPACE

```diff
@@ -3,6 +3,9 @@
 export(basic_table_args)
 export(clean_brushedPoints)
 export(draggable_buckets)
+export(formatters_var_labels)
+export(formatters_var_relabel)
+export(formatters_with_label)

```

Comment:
`var_labels<-` function is missing.

Reply:
yeah, I was wondering if we need that. thanks for pointing this up for teal.transform https://github.com/insightsengineering/teal.transform/pull/148#discussion_r1267642939
Just exported for teal.widgets in this commit 6d9fa1737e9f6d5a4eb07002a488d5a5469b542f